ezpublish/config/ezpublish_behat.yml

```diff
@@ -1,46 +1,2 @@
-doctrine:
-    dbal:
-        connections:
-            default:
-                driver: pdo_mysql
-                host: localhost
-                user: root
-                dbname: behattestdb
-                charset: UTF8
-
-ezpublish:
-    repositories:
-        behat: ~
-    siteaccess:
-        # Available siteaccesses
-        list:
-            - behat_site
```

Comment:
The tests run on the DemoBundle siteaccesses now. I haven't seen any reason why it wouldn't work.

Well, there is the fact that it is installed with German as the second language, but I haven't seen any language related tests, unless I'm mistaken.